test/integration.lean

```diff
@@ -37,6 +37,9 @@ example : ∫ x in 0..π, cos x ^ 2 - sin x ^ 2 = 0 := by simp [integral_cos_sq_
 /- the exponential function -/
 example : ∫ x in 0..2, -exp x = 1 - exp 2 := by simp
 
+/- the logarithmic function -/
+example : ∫ x in 1..2, log x = 2 * log 2 - 1 := by norm_num [sub_add]
```

Comment:
```suggestion
example : ∫ x in 1..2, log x = 2 * log 2 - 1 := by { norm_num, ring }
```
I think that this feels less magical (and it is a formulation that should work very often for the end user).